sidebar.c

```diff
@@ -355,25 +356,28 @@ static int cb_qsort_sbe(const void *a, const void *b)
 static void update_entries_visibility(void)
 {
   short new_only = C_SidebarNewMailOnly;
+  short non_empty_only = C_SidebarNonEmptyOnly;
```

Comment:
I know you're just doing as the previous line, but I really can't think of a reason for copying the boolean flag into a temporary short? Can we get rid of both?

Reply:
Agreed! I was about to mention that soon, once this is done. I'll fix it in a commit.

Reply:
My comment was more in favour of removing `new_only` and `non_empty_only` altogether. Why have them at all when they are non-modified copies of the `C_*` flags?

Reply:
Aah. To me, they're there because it makes the code a little easier to parse.

Reply:
Can we at least make them const, so one knows they're really just aliases and not going to change because of some logic some lines below?

Reply:
Agreed. Force pushed an updated commit